src/assets.rs

```diff
@@ -486,130 +528,155 @@ mod tests {
         use std::os::unix::ffi::OsStrExt;
 
         let test = SyntaxDetectionTest::new();
+        let config = Config::default();
 
         assert_eq!(
-            test.syntax_for_file_os(OsStr::from_bytes(b"invalid_\xFEutf8_filename.rs")),
+            test.syntax_for_file_os(OsStr::from_bytes(b"invalid_\xFEutf8_filename.rs"), &config),
             "Rust"
         );
     }
 
     #[test]
     fn syntax_detection_same_for_inputkinds() {
-        let mut test = SyntaxDetectionTest::new();
+        let test = SyntaxDetectionTest::new();
+        let mut config = Config::default();
+        config.syntax_mapping = SyntaxMapping::builtin();
 
-        test.syntax_mapping
+        config
+            .syntax_mapping
             .insert("*.myext", MappingTarget::MapTo("C"))
             .ok();
-        test.syntax_mapping
+        config
+            .syntax_mapping
             .insert("MY_FILE", MappingTarget::MapTo("Markdown"))
             .ok();
 
-        assert!(test.syntax_is_same_for_inputkinds("Test.md", ""));
```

Comment:
To make the diff nicer (i.e. easier to review) I would prefer if you kept the signature of `syntax_is_same_for_inputkinds()` and instead handled any new API inside of it. Possibly with a new wrapper for new tests.

But in general, I would like old regression tests to remain the same.

Reply:
This should be fixed with the latest update.